<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorithmExplorer</class>
 <widget class="QDockWidget" name="AlgorithmExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algorithms</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <widget class="QPushButton" name="btnRun">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbALgorithms"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeAlgorithms">
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
